cmd/agent/gui/views/templates/generalStatus.tmpl

```diff
@@ -498,6 +498,43 @@
     </span>
   </div>
 
+  <div class="stat">
+    <span class="stat_title">NetFlow</span>
+    <span class="stat_data">
+        Total Listeners: {{.netflowStats.TotalListeners}}
+        <br>Open Listeners: {{.netflowStats.OpenListeners}}
+        <br>Closed Listeners: {{.netflowStats.ClosedListeners}}
+        {{ if .netflowStats.OpenListeners }}
+        <br>
+        <span class="stat_subtitle">Open Listener Details</span>
+        {{- range $index, $NetflowListenerStatus := .netflowStats.WorkingListenerDetails }}
+        <br>BindHost: {{$NetflowListenerStatus.Config.BindHost}}
+        <br>FlowType: {{$NetflowListenerStatus.Config.FlowType}}
+        <br>Port: {{$NetflowListenerStatus.Config.Port}}
+        <br>Workers: {{$NetflowListenerStatus.Config.Workers}}
+        <br>Namespace: {{$NetflowListenerStatus.Config.Namespace}}
+        <br>Error: {{$NetflowListenerStatus.Error}}
+        <br>
+        {{- end }}
+        {{ end }}
+        <br>
+        {{ if .netflowStats.ClosedListeners }}
```

Comment:
Is there a need to separate the open/active and closed/unactive listeners?

I'm not sure this is bring that much value since the information from the two lists are very similar.
Also the closed listeners list is expected to be zero most of the them.